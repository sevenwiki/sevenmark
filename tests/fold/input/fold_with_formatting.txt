{{{#fold
[[**bold text** and normal]]
[[*italic text* here]]
}}}