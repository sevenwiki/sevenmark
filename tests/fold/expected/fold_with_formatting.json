[
  {
    "FoldElement": {
      "location": {
        "start": 0,
        "end": 64
      },
      "common_style": {},
      "content": [
        {
          "common_style": {},
          "content": [
            {
              "Bold": {
                "location": {
                  "start": 11,
                  "end": 24
                },
                "content": [
                  {
                    "Text": {
                      "location": {
                        "start": 13,
                        "end": 22
                      },
                      "content": "bold text"
                    }
                  }
                ]
              }
            },
            {
              "Text": {
                "location": {
                  "start": 24,
                  "end": 35
                },
                "content": " and normal"
              }
            }
          ]
        },
        {
          "common_style": {},
          "content": [
            {
              "Italic": {
                "location": {
                  "start": 40,
                  "end": 53
                },
                "content": [
                  {
                    "Text": {
                      "location": {
                        "start": 41,
                        "end": 52
                      },
                      "content": "italic text"
                    }
                  }
                ]
              }
            },
            {
              "Text": {
                "location": {
                  "start": 53,
                  "end": 58
                },
                "content": " here"
              }
            }
          ]
        }
      ]
    }
  }
]